realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -369,6 +379,19 @@ public void emit(JavaWriter writer) throws IOException {
         writer.endMethod();
     }
 
+    private void emitRealmInteger(JavaWriter writer, VariableElement field, String fieldName, String fieldTypeCanonicalName) throws IOException {
+        writer.emitAnnotation("Override");
+        writer.beginMethod(fieldTypeCanonicalName, metadata.getInternalGetter(fieldName), EnumSet.of(Modifier.PUBLIC))
+                .emitStatement("return new ManagedRealmInteger(0)")
+                .endMethod()
+                .emitEmptyLine();
+
+        writer.emitAnnotation("Override");
+        writer.beginMethod("void", metadata.getInternalSetter(fieldName), EnumSet.of(Modifier.PUBLIC), fieldTypeCanonicalName, "value")
+                .emitStatement("throw new IllegalStateException(\"Cannot assign to a managed RealmInteger\")")
```

Comment:
Not that I recall.  The conclusion over there was that assigning a managed object to an unmanaged object was allowed.